package.json

```diff
@@ -61,16 +61,19 @@
     "stylelint-order": "^0.8.1",
     "stylelint-scss": "^3.1.0",
     "stylelint-suitcss": "^3.0.0",
-    "terra-clinical": "git://github.com/cerner/terra-clinical.git#terra-clinical-onset-picker@2.4.0",
-    "terra-core": "git://github.com/cerner/terra-core.git#terra-toggle@2.8.0",
-    "terra-dev-site": "0.6.0",
-    "terra-framework": "git://github.com/cerner/terra-framework.git#terra-slide-panel-manager@2.5.0",
+    "terra-clinical": "git://github.com/cerner/terra-clinical.git#Consume-terra-dev-site-1.0",
+    "terra-core": "git://github.com/cerner/terra-core.git#consume-terra-dev-site-1.0",
+    "terra-dev-site": "^1.0.0",
+    "terra-framework": "git://github.com/cerner/terra-framework.git#consume-terra-dev-site-1.0",
     "terra-grid": "^4.0.1",
     "terra-heading": "^2.0.1",
     "terra-i18n": "^2.0.0",
     "terra-image": "^2.0.1",
     "terra-text": "^2.0.1",
     "terra-toolkit": "^3.0.0",
     "xfc": "^1.2.1"
+  },
+  "dependencies": {
```

Comment:
devdependency?

Reply:
These should all be dependencies except for the linters, but i'd like to do that in a different PR

Reply:
Issue logged: https://github.com/cerner/terra-ui/issues/92